tools/acrn-crashlog/common/fsutils.c

```diff
@@ -1014,7 +1014,7 @@ int find_file(const char *dir, size_t dlen, const char *target_file,
 
 int read_file(const char *path, unsigned long *size, void **data)
 {
-	char tmp[1024] = "\0";
+	char tmp[1024];
```

Comment:
I remember this is added by somebody in a separate patch to fix an issue. Now we want revert it. What is changed here?